plugins/templates/remove.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const baseSchema = schema.models.template.base;
+
+module.exports = () => ({
+    method: 'DELETE',
+    path: '/templates/{name}',
+    config: {
+        description: 'Delete a template',
+        notes: 'Returns null if successful',
+        tags: ['api', 'templates'],
+        auth: {
+            strategies: ['token'],
+            scope: ['user', '!guest']
```

Comment:
So only `user` can call this endpoint?
The permission is tied to the pipeline, and builds belong to that pipeline can publish, so I don't see why it's not allowed to delete?